include/internal/ktls.h

```diff
@@ -118,12 +118,13 @@ static ossl_inline int ktls_start(int fd,
 static ossl_inline int ktls_send_ctrl_message(int fd, unsigned char record_type,
                                               const void *data, size_t length)
 {
-    struct msghdr msg = { 0 };
+    struct msghdr msg;
     int cmsg_len = sizeof(record_type);
     struct cmsghdr *cmsg;
     char buf[CMSG_SPACE(cmsg_len)];
```

Comment:
the only issue here, is that buf is not guaranteed to be aligned.
I think in another PR, we started to make buf a union with a struct cmsghdr

Reply:
Definitely a different PR.  This one is deliberately restricted in scope.